files/default/neutron-ha-tool.py

```diff
@@ -646,6 +662,29 @@ def wait_router_ports_active(qclient, router_id, target_host, maxtries=60):
                            (router_id, ", ".join(remaining)))
 
 
+def ssh_delete_router_namespace(target_host, router_id):
+    namespace = "qrouter-" + router_id
+
+    ssh_client = paramiko.SSHClient()
+    ssh_client.set_missing_host_key_policy(paramiko.AutoAddPolicy())
+    ssh_client.load_system_host_keys()
+    ssh_client.connect(target_host)
+    stdin, stdout, stderr = ssh_client.exec_command("ip netns delete " +
+                                                    namespace)
```

Comment:
Ah right. In this case I wonder if neutron-ha-tool is the right place to put the code, instead in some cleaning code. Not sure about this.